sylvia/tests/ui/method_signature/reply.stderr

```diff
@@ -0,0 +1,19 @@
+error: Mismatched parameter in reply handlers.
```

Comment:
Ye, it's because of the limitation of [`proc-macro-error`](https://docs.rs/proc-macro-error/latest/proc_macro_error/) crate. Both spans are visible when run on `nightly`.